addons/l10n_fr_pos_cert/static/src/js/TicketScreen.js

```diff
@@ -0,0 +1,15 @@
+odoo.define('l10n_fr_pos_cert.TicketScreen', function(require) {
+    'use strict';
+
+    const TicketScreen = require('point_of_sale.TicketScreen');
+    const Registries = require('point_of_sale.Registries');
+
+    const PosFrCertTicketScreen = TicketScreen => class extends TicketScreen {
+        shouldHideDeleteButton(oder) {
+            return this.env.pos.is_french_country() || super.shouldHideDeleteButton(order);
```

Comment:
We figured out that we could delete if there was no payment methods and no orderlines. 
Can you introduce it there?

Reply:
Is it possible to have a POS without payment methods?

Reply:
@rhe-odoo